example/lib/main.dart

```diff
@@ -137,7 +144,7 @@ class _MyAppState extends State<MyApp> {
             SizedBox(height: 20),
             ElevatedButton(
               onPressed: () {
-                  signOut();
+                signOut();
```

Comment:
@danielgomezrico @cdmunoz Is this formatting change okay? I can remove it if needed.